core/context.go

```diff
@@ -286,14 +301,27 @@ func (ctx *Context) UpdateBackend(vsID, rsID string, weight int32) (int32, error
 	// Save the old backend weight and update the current backend weight.
 	result, rs.options.Weight = rs.options.Weight, weight
 
+	// Currently the backend options are changing only the weight.
```

Comment:
I partially agree here – what do you think should be the behavior if the user changes backend's weight in the store, from let's say 100 to 50 because we know that it can't take the full request load, and at the same time Gorb observes failed pulses and also tunes the weight down? Should we re-weight the backend maybe?